packages/inventory/src/containers/ListView/index.tsx

```diff
@@ -131,13 +141,23 @@ const ServicePointList = (props: ServicePointsListTypes) => {
       <Row className={'list-view'}>
         <Col className={'main-content'}>
           <div className="main-content__header">
+            <SearchForm {...searchFormProps} />
             <Link to={ADD_SERVICE_POINT}>
               <Button type="primary" size="large" disabled>
                 {ADD_SERVICE_POINT}
               </Button>
             </Link>
           </div>
-          <Table dataSource={dataSource} columns={columns}></Table>
+          <Table
+            dataSource={dataSource}
+            columns={columns}
+            pagination={{
+              showQuickJumper: true,
+              showSizeChanger: true,
+              defaultPageSize: 5,
+              pageSizeOptions: ['5', '10', '20', '50', '100'],
```

Comment:
Would it be better to move this options to constants?

Reply:
cool.